custom_components/tornado/__init__.py

```
@@ -7,19 +7,31 @@
 from typing import TYPE_CHECKING
 
 from homeassistant.const import Platform
+from homeassistant import config_entries
```

Ruff (0.8.2)

`homeassistant.config_entries` imported but unused

(F401)